CHANGES.rst

```diff
@@ -1,21 +1,32 @@
 Changes
 =======
 
-Version 0.8.1 (UNRELEASED)
+Version 0.8.1 (2022-02-09)
 --------------------------
 
 - Users:
     - Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
+    - Adds Kubernetes job memory limits validation before publishing workflow submission.
+    - Adds support for HTML preview of workspace files in the user interface.
+    - Adds search by name in workflow file list page of the user interface.
+    - Changes user interface to display a more accurate Cluster Health status page, based on the availability instead of the usage.
+    - Changes ``info`` command to include the list of supported compute backends.
+    - Fixes workflow stuck in pending status due to early Yadage failures.
 - Administrators:
     - Adds configuration to set default job timeout value for the Kubernetes compute backend jobs (``kubernetes_jobs_timeout_limit``).
     - Adds configuration to set maximum job timeout that users can assign to their jobs for the Kubernetes compute backend (``kubernetes_jobs_max_user_timeout_limit``).
     - Adds new configuration option ``compute_backends`` to specify the supported list of compute backends for validation purposes.
     - Adds new configuration option ``reana_server.uwsgi.log_all`` to toggle the logging of all the HTTP requests.
     - Adds new configuration options ``reana_server.uwsgi.log_4xx`` and ``reana_server.uwsgi.log_5xx`` to only log HTTP error requests, i.e. HTTP requests with status code 4XX and 5XX. To make this configuration effective ``reana_server.uwsgi.log_all`` must be ``false``.
     - Adds new configuration options ``node_label_infrastructuremq`` and ``node_label_infrastructuredb`` to have the possibility to run the Message Broker and the Database pods in specific nodes.
+    - Adds workflow name validation to the ``create_workflow`` endpoint, restricting special characters like dots.
```

Comment:
This news item and the similar ones regarding AP may be considered for "developers" rather than "administrators". The regular cluster admins usually care about deploying and configuring REANA, so Helm, uWSGI etc, but they may not care about API endpoints at all. That's more for application developers in my eyes. What about moving those to the new "Developers" section?